api/src/test/resources/underlays/nautical.pbtext

```diff
@@ -0,0 +1,305 @@
+# proto-file: proto/bio/terra/tanagra/underlay/underlay.proto
```

Comment:
Another thought: yaml may be a more accessible / common config format here. YAML can be converted to JSON, which can be converted to proto using the proto3 JSON encoding, so in theory that pathway could also work with little additional effort.

Reply:
Added an equivalent yaml file and configuration property to allow it.

Reply:
Thanks! Is it pushed? I'm not seeing it in the branch

Reply:
Merge error, ptal now!